compile_commands.json

```diff
@@ -1,127 +0,0 @@
-[
```

Comment:
Please stop committing this every time. Instead of running `git add .`, use the VS code menu to select which files to commit.